### DISCUSSION TITLE
Setup with Docker (Simple) fails with Database could not be configured

### DISCUSSION THREAD

**Discussion by don-ferris** [Kovah/LinkAce#707] [Support]
Followed the procedure [here](https://www.linkace.org/docs/v1/setup/setup-with-docker/simple/) 
Went to the proper URL and got "Welcome to the LinkAce setup"
Checked requirements (all good - Continue)
Left Database Configuration at defaults (I had already filled in the db details in the .env file) and clicked Configure Database...

**Database could not be configured. Please check your connection details. Details: SQLSTATE[HY000] [1045] Access denied for user 'linkace'@'172.27.0.3' (using password: YES) (SQL: SHOW FULL TABLES WHERE table_type = 'BASE TABLE')**

I tried setting permissions (recursively) for /var/lib/mysql to 777 - no help

**Comment by don-ferris** [comment]
Additional details:
The DB password I used was 16 characters with no special characters.
The only other thing I changed was the port number for accessing the LinkAce app.

**Comment by mattisaoldgit** [comment]
I am getting the same, things I did that failed to resolve it:
Each test was using either simple or complex docker and each time containers were deleted

- Made sure no special chars in password
- Put passwords in docker-compose instead of .env
- Made sure .env had rw permissions

I can't figure out how to connect to the running DB and see whats up, I suspect maybe is a IP restriction thing, but dont know how to fix even if it is that.

Any ideas anyone, I would really like to give this a try :)

Thanks, you are lovely for even reading this far :)

**Reply by mattisaoldgit** [reply]
I got it working, not quite sure why, here is what I added to the docker-compose.yml

```
 - MARIADB_ROOT_PASSWORD=RootPass
 - MARIADB_USER=linkace
 - MARIADB_PASSWORD=ExamplePass
 - MARIADB_DATABASE=linkace
 

```
TIP: an easy way to clear things down after a failed attempt is to type this:
`docker-compose down --volumes`
It will "reset" the docker so your new settings are used and there are no conflicts (yes I know that's not entirely true)

I hope that helps
